BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m

```diff
@@ -72,13 +72,13 @@ + (CGFloat)convertPointsWidthToPercentage:(float) width forWindow:(UIWindow*)win
 
 + (CGFloat)convertPercentageHeightToPoints:(float) height forWindow:(UIWindow*)window {
     CGFloat presentationAreaHeight = [self getPresentationAreaHeightForWindow:window];
-    CGFloat heightInPoints = (CGFloat) ceil(presentationAreaHeight * (height / 100.0f));
+    CGFloat heightInPoints = (CGFloat) round(presentationAreaHeight * (height / 100.0f));
```

Comment:
What is the difference between these two methods?

Reply:
convertPercentageHeightToPoints is converting the given in-app percentage height to points height and
convertPercentageWidthToPoints is converting the given in-app percentage width to points width

Reply:
No, I was asking about the ceil and round functions. :) I guess ceil rounds everything up to the next decimal where round only rounds if the decimal value is .5 or above.

Reply:
Yes, that's correct.